_posts/Installation/0000-01-01-standard_install.md

```diff
@@ -1,24 +1,26 @@
 ---
-title: "LibrePhotos Install"
-excerpt: "How to install Libre Photos."
+title: "Docker Compose"
+excerpt: "How to install Libre Photos using Docker Compose."
 last_modified_at: 2021-03-07
 category: 1
 ---
 
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=LibrePhotos_ownphotos&metric=alert_status)](https://sonarcloud.io/dashboard?id=LibrePhotos_ownphotos) ![Discord](https://img.shields.io/discord/784619049208250388?style=plastic) ![Website](https://img.shields.io/website?down_color=lightgrey&down_message=offline&style=plastic&up_color=blue&up_message=online&url=https%3A%2F%2Flibrephotos.com) ![GitHub contributors](https://img.shields.io/github/contributors/librephotos/librephotos?style=plastic)
 
-**Currently the project is in very early stages, some bugs may exist. If you find any please log an issue**
+> Currently the project is in very early stages; some bugs may exist. If you find any please log an issue
 
 ### Compatibility
+
 You need a x86 processor and it is recommended to have 8GBs of RAM.
 
-**It kind of works on ARM processors. Due to limited ram & processing power it will not be as fast.**
+#### ARM
 
-**ARM Development is still in progress expect bugs.**
+You can run this on ARM using the ARM compose templates. Due to limited ram & processing power it will not be as fast. ARM Development is still in progress expect bugs.
```

Comment:
```suggestion
You can run this on ARM using the ARM compose templates. Due to limited ram & processing power it will not be as fast. ARM Development is still in progress - expect bugs.
```
Grammar